src/index.js

```diff
@@ -175,21 +179,56 @@ class GracefulImage extends Component {
             };
           });
         }, this.state.retryDelay * 1000);
+      } else {
+        this.setState({ fallbackImage: this.props.fallbackImage });
       }
     });
   }
 
   /*
     - If image hasn't yet loaded AND user didn't want a placeholder OR SVG not supported then don't render anything
-    - Else if image has loaded then render the image
-    - Else render the placeholder
+    - Else if image has not loaded and fallback image is given, then render the fallback image
+    - Else render the default placeholder until image is loaded.
   */
   render() {
     if (!this.state.loaded && (this.props.noPlaceholder || !IS_SVG_SUPPORTED))
       return null;
+    else if (!this.state.loaded && this.state.fallbackImage) {
+      const style = {
+          animationName: "gracefulimage",
+          animationDuration: "0.3s",
+          animationIterationCount: 1,
+          animationTimingFunction: "ease-in",
+          transform: 'translateY(50%)',
+        };
+      const wrapperStyle = {
+        width: this.props.width,
+        height: this.props.height,
+        backgroundColor:'#f2f3f4',
+        textAlign:'center'
+      };
+      
+      return (
+        <div style={{ ...wrapperStyle }}>
+          <img
+            src={this.state.fallbackImage}
+            className={this.props.className}
+            width='65px'
+            height='65px'
+            style={{
+              ...style,
+              ...this.props.style
+            }}
+            alt={this.props.alt}
+            ref={this.state.loaded ? null : ref => (this.placeholderImage = ref)}
+          />
+          <div style={{ fontSize:'14px', color:'#c1c7c9', paddingTop:'40px' }} >{this.props.fallbackMessage}</div>
```

Comment:
Maybe the styles for the message should be left up to the consumer. The fallbackMessage could be `PropTypes.node` or `PropTypes.element`?

Reply:
sure I think that's good too